docs/conf.py

```diff
@@ -262,15 +256,11 @@ def rstjinja(app, docname, source):
 # JSOC email os env
 os.environ["JSOC_EMAIL"] = "jsoc@cadair.com"
 
-# JSOC email os env
```

Comment:
How was this here twice?

Reply:
That may have been my fault. I added it in the cutout PR and it was also in the metadata PR. Maybe the rebase got screwed up somehow.

Reply:
Ah, all I did was merge in master so that would explain it.